bedrock/firefox/tests/test_views.py

```diff
@@ -634,6 +634,38 @@ def test_firefox_home(self, render_mock):
         assert template == ["firefox/home/index-master.html"]
 
 
+class TestFirefoxGA(TestCase):
+    def assert_ga_attr(self, response):
+        cta_types = ["link", "button"]
+        doc = pq(response.content)
+        links = doc(".mzp-c-button")
+        for link in links.items():
+            cta_data = link.attr("data-cta-type")
+            cta_link = link.attr("data-link-type")
+            if cta_data:
+                contain_cta = any(cta_data in list for list in cta_types)
+                assert contain_cta or "fxa-" in cta_data
+            elif cta_link:
+                # some links with the data-link-type use button instead of download which is what is described
+                # in the docs, so not sure if we should allow this or not
+                assert cta_link == "download" or cta_link == "button"
```

Comment:
Suggestion (non-blocking): doing a search for `data-link-type` I see several variations of possible values including `link`, `button`, `download`, `nav`, `footer`.

I think we can probably ignore `nav` and `footer` here as these are not main CTA links. So I guess we should check the following:

```suggestion
                cta_link_types = ["button", "link", "download"]
                assert cta_link in cta_link_types
```

Thoughts?